system/autoware_version/src/autoware_version.cpp

```diff
@@ -41,12 +42,21 @@ class AutowareVersionNode : public rclcpp::Node
     RCLCPP_INFO(get_logger(), "ROS_DISTRO=%s", message.ros_distro.c_str());
 
     publisher_ = create_publisher<autoware_system_msgs::msg::AutowareVersion>(
-      "autoware_version", rclcpp::QoS{1}.transient_local());
-    publisher_->publish(message);
+      "autoware_version", rclcpp::QoS{1});
+
+    auto callback = [this, message] { publisher_->publish(message); };
```

Comment:
Could you add a `TODO` comment saying that this is just a temporary fix until rosbridge supports transient_local QoS.

Reply:
Added.